packages/lib-react-components/test/setup.js

```diff
@@ -13,7 +13,7 @@ chai.use(sinonChai)
 global.React = React
 global.expect = chai.expect
 
-const jsdom = new JSDOM('<!doctype html><html><body></body></html>')
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>', { url: 'https://localhost'})
```

Comment:
What does this addition change?

Reply:
Without it, `Markdownz` throws security errors that fail the tests. Do you know why that might be happening?

Reply:
Hmm no I'm not sure. I'm not very familiar with test environment setup.